modules/ROOT/pages/database-administration/index.adoc

```diff
@@ -1,5 +1,5 @@
 :description: This page gives an introduction to database administration using Neo4j.
-:page-aliases: manage-databases/configuration, manage-databases/introduction
+:page-aliases: manage-databases/configuration.adoc, manage-databases/introduction.adoc
```

Comment:
I can do that, but we shouldn't remove `manage-databases/configuration`from the aliases because it is giving 404, unless you believe it's better to redirect to another page?

Reply:
Yes, it should be an alias of several other pages. I can look into them and add them to the card if you want to work on it.

Reply:
This is the card I am talking about https://trello.com/c/ex0r3y2n/217-check-if-all-page-aliases-are-backed-by-a-redirect

Reply:
manage-databases/configuration is not an alias of the database administration page

Reply:
yes, if you can see what the other pages need to have alias fixed, we can use this same pr. also, do you have a suggestion for another redirect to manage-databases/configuration?

Reply:
I have to look into it. That's why I created this card. This came up after Niel's clarification about the use of aliases.

Reply:
ok, so I suppose it's better to merge this one now and then you create another one later like you said :)